tests/product.test.js

```diff
@@ -5,12 +5,18 @@ const { models, connectDb } = require ('../dal/database');
 // dummy documents
 const products = [{
   _id: new ObjectID(),
-  product: 'mobile phone',
-  manufacturers: ["Samsung", "AT&T", "LG Electronics"]
+  barcodeId: "8152210266539",
+  name: "Macbook Pro 2018",
+  category: "laptop",
+  manufacturer: "Apple",
+  ESG: "1"
```

Comment:
Please change scale to 0-100 (so esg = 100 in this case)

Reply:
Will make changes once I write unit tests for the routes.